### ISSUE TITLE
point.events.click not fired during simulation in packedBubble series when clicking on HTML dataLabel

### ISSUE THREAD

**Issue by weixingcai** [highcharts/highcharts#13658]
The click event of the Events event should be triggered when the ball is clicked

https://jshare.com.cn/temp/tBG0z1
![1](https://user-images.githubusercontent.com/48822299/83997873-fdb3ea00-a991-11ea-8989-3d2c71e188ec.jpg)

**Comment by weixingcai** [comment]
Where I marked the red box there was a click event, where I hit the label, but it didn't fire, but in the ball without the label, it fired

**Comment by Izothep** [comment]
Hi @weixingcai,

Thank you for contacting us with your issue.

In current `packedbubble` series version there is a limitation of triggered click events, related to unfinished simulation. It is possible to trigger click event of point label but only after graph simulation is finished.

As a workaround, it is possible to disable `enableSimulation` in `layoutAlgorithm` option.

FYI: @mateuszkornecki

**Comment by weixingcai** [comment]
No animation effect, it is not the one I need, can only be solved like this now

**Comment by weixingcai** [comment]
You can click when you don't have tags or when you don't have HTML tags

**Comment by Izothep** [comment]
Thank you for sending further information about the problem. Simplified demo:
https://jsfiddle.net/BlackLabel/0456rjht/1/

**Comment by pawelfus** [comment]
Let me rephrase the title, okay @weixingcai ?

Did you try setting `useHTML: false`? Could you let me know the results? Thanks!

**Comment by weixingcai** [comment]
However, what I need is the case of useHTML: true, And I need to define the required tags myself, so this problem will occur. This does not happen if it is false

**Comment by pawelfus** [comment]
Okay, thanks for the confirmation!

**Comment by weixingcai** [comment]
UseHTML: true is there no way to solve that problem when using HTML: true

**Comment by pawelfus** [comment]
I don't see any other workaround: either disable simulation or `useHTML` option.

**Comment by weixingcai** [comment]
Do you optimize things like tha

**Comment by pawelfus** [comment]
We fix issues like that. No ETA for this ticket, though.

**Comment by weixingcai** [comment]
thanks

**Comment by pawelfus** [comment]
Fixed since v9: https://jsfiddle.net/BlackLabel/stzxv4ko/
